server/routes/shared/primaryNavBar/primaryNavBarPresenter.ts

```diff
@@ -0,0 +1,44 @@
+import { PrimaryNavItem } from '../../../utils/viewUtils'
+import config from '../../../config'
+import LoggedInUser from '../../../models/loggedInUser'
+
+export default class PrimaryNavBarPresenter {
+  constructor(private readonly active: string, private readonly loggedInUser: LoggedInUser) {}
+
+  // populate the nav bar items based on user roles
+  get items(): PrimaryNavItem[] {
+    const items = []
+
+    if (this.loggedInUser.token.roles.includes('ROLE_CRS_PROVIDER')) {
```

Comment:
Would also be nice to have a Decorator(? *) to obtain role information.

*at least that's what we named and use in other places i.e AppointmentDecorator

i.e. 

`if(userDecorator.hasCrsRole)`

Reply:
do you think this is any clearer than the above? i think perhaps it obfuscates what the actual role->content mapping is?

Reply:
oh I was just thinking that we might perhaps need to do this in several places in future.

Maybe leave it until that is the case